Podpeople available as roundstart ft. minor stat changes

Stokeling9701 opened the pull request:
Essentially it allows podmen to be roundstart along with editing them so they take slightly more fire damage and more damage from plant-b-gone

 :cl: Daniel0Mclovin
add: Podpeople to roundstart
tweak: Modified podpeople heatmod from 1.25 to 1.5
tweak: plant-b-gone now does roughly 7 damage per tick on podpeople
/:cl:

[Reason]:
It'd be nice to see pod people around a bit more. Opens up more roleplay opportunities and another creature to see on board. Only supports basic color changes.

Stokeling9701 committed:
Podpeople as roundstart

Essentially it allows podmen to be roundstart along with editing them so they take slightly more fire damage and more damage from plant-b-gone

Wilchenx commented on code/modules/mob/living/carbon/human/species_types/podpeople.dm:
```diff
@@ -10,8 +10,11 @@
-	liked_food = NONE
+	disliked_food = MEAT
+	liked_food = VEGETABLES | FRUIT
+	toxic_food = TOXIC | RAW
```
Citadel just changed it to none in another PR

Stokeling9701 replied:
ah i didnt know that, why was it changed?

Ratvar replied:
"stealthly nerfed lizrds cannot eat donk pockets" blahblah

silicons replied:
set to none

silicons commented on code/modules/mob/living/carbon/human/species_types/podpeople.dm:
```diff
@@ -10,8 +10,11 @@
+	disliked_food = MEAT
+	liked_food = VEGETABLES | FRUIT
+	toxic_food = TOXIC | RAW
+	roundstart = 1
```
TRUE

silicons commented on code/modules/mob/living/carbon/human/species_types/podpeople.dm:
```diff
@@ -38,14 +41,17 @@
+		H.adjustToxLoss(5)
 		H.reagents.remove_reagent(chem.id, REAGENTS_METABOLISM)
 		return 1
+		H.confused = max(H.confused, 1)
```
make this above the return

silicons commented on code/modules/mob/living/carbon/human/species_types/podpeople.dm:
```diff
@@ -38,14 +41,17 @@
-		H.adjustToxLoss(3)
+		H.adjustToxLoss(5)
 		H.reagents.remove_reagent(chem.id, REAGENTS_METABOLISM)
 		return 1
```
and while your'e at it, return TRUE

Stokeling9701 committed:
changed a few 1's to TRUE's, removed food prefs

deathride58 reviewed (commented):
You should probably nerf the heal_overall_damage and adjustToxLoss in spec_life to around 0.05. Keeping them at TG's value of 1 would mean podpeople wouldn't have to take a single trip to medbay in most cases. Nerfing their passive regeneration to 0.05 would mean antags won't have to break a bunch of lights just to crit a podperson in hand-to-hand combat, and it'd mean podpeople would actually have to visit medbay if they stay in the dark too long.

Stokeling9701 committed:
Changed a few stats

Darkness now does oxy damage and slight toxin, added a bit more heat mod, no longer heals toxin damage, oxygen loss healing nerfed. Can eat in the dark to try to combat starvation, added slight toxin damage to make it harder to stay in the dark with a shit ton of food.

Stokeling9701 committed:
Nerfed overall healing

dialed down to 0.7

Stokeling9701 committed:
Accidentally put a comma

silicons commented on code/modules/mob/living/carbon/human/species_types/podpeople.dm:
```diff
@@ -34,18 +37,21 @@
+			H.adjustOxyLoss(-0.5)
+
+	if(H.nutrition < NUTRITION_LEVEL_STARVING + 55)
+		H.adjustOxyLoss(5,5) //can eat to negate this unfortunately
```
are you sure this is how you use oxygen damage?

silicons commented on code/modules/mob/living/carbon/human/species_types/podpeople.dm:
```diff
@@ -34,18 +37,21 @@
+
+	if(H.nutrition < NUTRITION_LEVEL_STARVING + 55)
+		H.adjustOxyLoss(5,5) //can eat to negate this unfortunately
+		H.adjustToxLoss(3,0)
```
are you sure this is how you use toxins damage?

silicons commented on code/modules/mob/living/carbon/human/species_types/podpeople.dm:
```diff
@@ -34,18 +37,21 @@
-			H.heal_overall_damage(1,1)
-			H.adjustToxLoss(-1)
-			H.adjustOxyLoss(-1)
+			H.heal_overall_damage(0.7)
```
are you sure this is how you use heal overall damage?

Stokeling9701 committed:
cleaned up some things

Stokeling9701 committed:
Edited regen

Made it proper

silicons reviewed (approved):
Code: Working, probably good, I'm assuming you tested all this.
Balance except for regen values: Good I guess.
REGEN VALUES: Imo contrary to the haters having to wait 40 seconds to heal a single point of damage? Why is it even there at that point, but whatever. I'd recommend a system that lets them heal faster (0.5-1.5) if they haven't taken damage in <x> amount of time but I don't care on that.
